ember/app/ui/components/identity-addresses/component.js

```diff
@@ -21,24 +22,40 @@ export default class IdentityAddressesComponent extends Component {
   @service notification;
   @service store;
   @service intl;
-  // @service can;
 
   // Lifecycle
 
-  @action onUpdate() {
+  @action
+  onUpdate() {
+    this.fetchCountries.perform();
     this.fetchAddresses.perform(this.args.identity);
   }
 
   // List
 
   @lastValue("fetchAddresses") addresses;
-
-  @restartableTask *fetchAddresses(identity) {
+  @restartableTask
+  *fetchAddresses(identity) {
     return yield this.store.query("address", {
       filter: { identity: identity.id },
     });
   }
 
+  @lastValue("fetchCountries") countries;
+  @restartableTask
+  *fetchCountries() {
+    const adapter = this.store.adapterFor("identity");
+    adapter.headers["Accept-Language"] =
+      window.localStorage.getItem("locale") ?? "en";
```

Comment:
No `window.` needed to access localStorage.

Reply:
Removed